src/lib.rs

```diff
@@ -16,14 +16,14 @@
 // Platform dependent whether necessary
 #![allow(clippy::unnecessary_cast)]
 // Lints
-#![deny(
-    missing_docs,
-    missing_copy_implementations,
-    trivial_casts,
-    unstable_features,
-    unused_import_braces,
-    unused_qualifications
-)]
+// #![deny(
+//     missing_docs,
+//     missing_copy_implementations,
+//     trivial_casts,
+//     unstable_features,
+//     unused_import_braces,
+//     unused_qualifications
+// )]
```

Comment:
No! Please don't comment out the lints!!!  This is to keep me honest and also to make sure PR's like this live up to the standards of the library. It seems that yours builds fine with the lints turned on, so please restore this.

Reply:
I'll be more careful about removing changes I made while debugging my code and about copy paste noise next time. Apologies.